src/nidcpower/metadata/functions_addon.py

```diff
@@ -2,18 +2,12 @@
 # Changes to the metadata should be made here, because functions.py is generated thus any changes get overwritten.
 
 functions_override_metadata = {
-    # TODO (ni-jfitzger): delete this when there's no difference from functions.py content
-    'SetRuntimeEnvironment': {
-        'codegen_method': 'private',
-        'method_templates': [
-            {
-                'documentation_filename': 'none',
-                'library_interpreter_filename': 'default_method',
-                'method_python_name_suffix': '',
-                'session_filename': 'none'  # will (intentionally) keep this method out of session.py, despite the codegen_method being private
-            }
-        ],
-    }
+    'ConfigureLCRCompensation': {
```

Comment:
The nipkg installer for ni-grpc-device-server shipping with the 2023 Q2 release of NI-DCPower does not have these entrypoints, because:

* the internal metadata had them turned off for a long time (until I re-enabled them for this change)
* we really lag behind in updating grpc-device metadata
   * the process is a bit painful and the grpc-device repo needs to be updated before release to make it into the nipkg.
   
   For these reasons, I have turned these entrypoints off in GrpcStubInterpreter until they are added to grpc-device-server.

Reply:
TODOs added, referencing #1920